src/reducer/admin/notification.js

```diff
@@ -6,12 +6,17 @@ import {
 const defaultState = {
     text: '',
     type: 'info', // one of 'info', 'confirm', 'warning'
+    autoHideDuration: undefined, // undefined means use global duration
 };
 
 export default (previousState = defaultState, { type, payload }) => {
     switch (type) {
         case SHOW_NOTIFICATION:
-            return { text: payload.text, type: payload.type };
+            return {
+                text: payload.text,
+                type: payload.type,
+                autoHideDuration: payload.autoHideDuration,
+            };
         case HIDE_NOTIFICATION:
             return { ...previousState, text: '' };
```

Comment:
I think this misses a `autoHideDuration: undefined`, otherwise a single notification with a specific duration will override the duration of all following notifications.

Reply:
nope, because every call of showNotification action set's new value or undefined
same as type